pkg/drawio/treeNodeInterface.go

```diff
@@ -27,7 +27,7 @@ package drawio
 // the structs abstractSquareTreeNode, abstractIconTreeNode, abstractLineTreeNode contains abstractTreeNode
 // All structs representing a Square (VpcTreeNode, ZoneTreeNode, SubnetTreeNode...) contains abstractIconTreeNode
 // All structs representing an icons (NITreeNode, GatewayTreeNode, UserTreeNode...) contains abstractIconTreeNode
-// All structs representing a line (VsiLineTreeNode, ConnectivityTreeNode) contains abstractIconTreeNode
+// All structs representing a line (LogicalLineTreeNode, ConnectivityTreeNode) contains abstractIconTreeNode
```

Comment:
```suggestion
// All structs representing a line (LogicalLineTreeNode, ConnectivityTreeNode) contain abstractLineTreeNode
```
Please review the comments above as well

Reply:
done

Reply:
I don't see it fixed. Can you please recheck?

Reply:
fixes